tasks/testwasher.py

```diff
@@ -88,3 +101,37 @@ def process_module_results(self, module_results: List[ModuleTestResult]):
             print(failed_tests_string)
 
         return should_succeed
+
+    def is_known_flaky_test(self, failing_test, known_flaky_tests, known_flaky_tests_parents):
+        """
+        Check if a test is known to be flaky
+        The method should be called with the following arguments:
+        - failing_test: the test that is failing
+        - known_flaky_tests: the set of tests that are known to be flaky
+        - known_flaky_tests_parents: the set of tests that are known to be flaky with their parents
```

Comment:
```suggestion
        - known_flaky_tests_parents: the set of tests that are ancestors of a known flaky test, thus would fail when the flaky leaf test fails
```